src/android/toga_android/widgets/scrollcontainer.py

```diff
@@ -55,7 +55,7 @@ def create(self):
         self.hScrollListener.is_scrolling_enabled = self.interface.horizontal
         self.hScrollView.setOnTouchListener(self.hScrollListener)
         vScrollView.addView(self.hScrollView, hScrollView_layout_params)
-        if self.interface.content is not None:
+        if self.interface.content:
```

Comment:
AFAICT, this one is OK - `self.interface.content` is a Python-side object, so comparisons with None are fine.